template/FullStack/Vue(Frontend)+Django(Backend)/server/setup.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)